systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -216,6 +217,42 @@ void testUserWithQuotas(KafkaUser user) {
         });
     }
 
+    @Test
+    void testCreatingUsersWithSecretPrefix() {
+        String clusterName = "second-cluster";
```

Comment:
Is there any first cluster?

Reply:
Yes, here:
```
private void deployTestSpecificResources() {
        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 1, 1).done();
    }
```
That's for other tests, so I guess it's better to have different kafka cluster here.

Reply:
Soooo ... I can live with it if that is really what you want to do. But this is kinda mess to create one cluster before all here, the next one inside test etc. It makes the tests hard to read and understand but also debug. Probably outside the scope of this PR, but it would be good to have this done in some more clear way and maybe not mix the tests in one class etc.

Reply:
I'll take a look at it and try to separate it to two suites, if it'll make sense. Thanks for this spot.

Reply:
Anyway, as I said, that is more general readability ... feel free to leave it for later, it does not need to be addressed in this PR.